opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -56,7 +69,9 @@ class UniformXTabulated2DFunction
     typedef std::tuple</*x=*/Scalar, /*y=*/Scalar, /*value=*/Scalar> SamplePoint;
 
 public:
-    UniformXTabulated2DFunction()
+
+    explicit UniformXTabulated2DFunction(const InterpolationGuide iGuide)
```

Comment:
It was removed since you must give a policy. Now that the change is done it should be possible to re-introduce the default constructor by using Vertical as default policy, but I think that runs the risk of this class being used carelessly: despite the rather general name, it has very specific uses with PVTO/PVTG tables.

If you insist I'll add the default argument, but I prefer not.

Reply:
I'd much prefer to have a default constructor, even if this means to create an object in undefined state because this object can later be assigned to:

```c++
UniformXTabulated2DFunction foo;
if (a > b)
 foo = bar;
else
 foo = baz;
```

I don't insist on it though...

Reply:
> I don't insist on it though...

We can revisit this if a need turns up.

Reply:
> We can revisit this if a need turns up.

it will not turn up: people will hack their way around it ;)